packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -156,43 +156,43 @@ This is Twig implementation of the [Accordion] component.
 
 ## API
 
-The Accordion itself consists of many components which cannot be used independently.
+The Accordion itself consists of several components which cannot be used independently.
 
 ### Accordion
 
-| Prop name     | Type     | Default     | Required | Description              |
-| ------------- | -------- | ----------- | -------- | ------------------------ |
-| `id`          | `string` | `null`      | no/yes   | Optional, depends on ... |
-| `class`       | `string` | `null`      | no       | Additional class name    |
-| `elementType` | `string` | `'section'` | no       | Custom element type      |
+| Prop name     | Type     | Default     | Required | Description                                                                                           |
+| ------------- | -------- | ----------- | -------- | ----------------------------------------------------------------------------------------------------- |
+| `id`          | `string` | `null`      | no/yes   | Optional, It depends on whether the "Always open" option is used, if so, the id field is not required |
```

Comment:
I am suggesting some changes in copy.
- `Always open` -> `Stay open` - we are using it at the beginning of this readme
- `option` -> `functionality` - there is no "option" for this, I think that "functionality" better describes it
- `yes/no` -> `optional` - not sure about this, but seems better to me. Please decide yourself.

```suggestion
| `id`          | `string` | `null`      | optional   | It depends on whether the "Stay open" functionality is used. If so, the id field is not required. |
```

Reply:
ok, `optional` is also good for me, fixed